package.json

```diff
@@ -60,6 +60,16 @@
   ],
   "main": "./out/src/extension.js",
   "contributes": {
+    "configuration": {
+      "type": "object",
+      "title": "Apache Camel Tooling Support configuration",
```

Comment:
I don't like that name tbh. It is much too long. How about something simple like "Apache Camel Tooling". I mean configuration is clearly obvious because thats what you do in the preferences, right? Maybe you have other ideas too.